frontend/index.html

```diff
@@ -274,6 +275,30 @@ <h4 class="modal-title"><span class="glyphicon glyphicon-cog"></span> Settings</
             </div>
         </div>
     </div>
+	
+	<!-- Defines the modal used for updating tags -->
+	 <div id="tags_update_div" class="modal fade tags-update-modal" tabindex="-1" role="dialog" aria-labelledby="tags-update-modal">
+        <div class="modal-dialog modal-lg">
```

Comment:
I think the tag update modal can be smaller by using `modal-sm` or leaving `modal-lg` out, because it does not need the full assigned width.